### ISSUE TITLE
Your Teams cards list

### ISSUE THREAD

**Issue by davidbenque** [cryptpad/cryptpad#1193]
from different parts of the audit

> The cards appearing under ‘Your Teams’ are visually list elements and should be coded as such. 

----

>The link ‘Open’ on each Team card needs a unique link texts. Blind users often ask for a list of links and it would help them if they know straight away which Team they will open with which link. For instance, you could add the name of the team to the link text and visually hide this extra information. 

comments below were probably made before changes in the card list? we should check that we have adhered to these suggestions

> Because you mentioned you are going to make the whole area clickable, please be aware of some common errors. A common mistake is to make different components that link to the same page clickable separately. This is annoying for users that need to use a keyboard, because they have to tab through 3 separate links instead of one. Another common error is to use one link and put all subcomponents (image, spans and link descriptions) inside the anchor tags. Because the card contains little text, it might not create big issues here, but be aware that this means a user of speech software might have trouble activating the link because of the amount of text attached to it, and it also creates long link texts for blind users. The best practice is to have one link (and programmatically give this link a clear link text) and use JavaScript to enlarge the clickable area.
